Portfolio/index.html

```diff
@@ -1,22 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
     <link rel="stylesheet" href="style.css">
     <title>Document</title>
-    <style> 
-    /* navbar */
+    <style>
+    /* navigation Bar */

```

Comment:
The 'position: relative;' property is floating outside of any selector or rule. It should be placed within a proper CSS selector.